docs/data-sources/scope.md

```diff
@@ -31,7 +31,7 @@ data "boundary_scope" "project" {
 ### Required
 
 - `name` (String) The name of the scope to retrieve.
-- `scope_id` (String) The parent scope ID that will be queried for the scope.
+- `parent_scope_id` (String) The parent scope ID that will be queried for the scope.
```

Comment:
I don't know what the implications are when changing something like this (since... I think it would break current implementations?). What's usually the guidance for something like this?